src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -365,39 +371,44 @@
-     (when (and (not no-routes-found?) (or loading-routes? route))
+     (when not-enough-asset?
+       [not-enough-asset])
+     (when (or (and (not no-routes-found?) (or loading-routes? route))
+               not-enough-asset?)
        [estimated-fees
         {:loading-routes? loading-routes?
          :fees            fee-formatted
          :amount          amount-text}])
-     (cond
-       show-no-routes?   [no-routes-found]
-       not-enough-asset? [not-enough-asset])
+     (when show-no-routes?
+       [no-routes-found])
      [quo/bottom-actions
       {:actions          :one-action
        :button-one-label (if should-try-again?
                            (i18n/label :t/try-again)
                            button-one-label)
        :button-one-props (merge (when-not should-try-again?
                                   button-one-props)
-                                {:disabled? (or loading-routes?
-                                                (and (not should-try-again?) confirm-disabled?))
-                                 :on-press  (cond
-                                              should-try-again?
-                                              #(rf/dispatch [:wallet/start-get-suggested-routes
-                                                             {:amount        amount-in-crypto
-                                                              :updated-token token-by-symbol}])
-                                              sending-to-unpreferred-networks?
-                                              #(show-unpreferred-networks-alert on-confirm)
-                                              :else
-                                              #(on-confirm amount-in-crypto))}
+                                {:disabled?           (or not-enough-asset?
+                                                          loading-routes?
+                                                          (and (not should-try-again?)
+                                                               confirm-disabled?))
+                                 :on-press            (cond
+                                                        should-try-again?
+                                                        #(rf/dispatch [:wallet/start-get-suggested-routes
+                                                                       {:amount        amount-in-crypto
+                                                                        :updated-token token-by-symbol}])
+                                                        sending-to-unpreferred-networks?
+                                                        #(show-unpreferred-networks-alert on-confirm)
+                                                        :else
+                                                        #(on-confirm amount-in-crypto))
+                                 :customization-color current-color}
```

Comment:
Added the missing customization color for this button. The customization color is the one picked for the wallet account.